DigitalOffice/Services/UserService/Mappers/EditUserRequestToDbMapper.cs

```diff
@@ -0,0 +1,34 @@
+﻿using System;
+using System.Security.Cryptography;
+using System.Text;
+using LT.DigitalOffice.UserService.Database.Entities;
+using LT.DigitalOffice.UserService.Mappers.Interfaces;
+using LT.DigitalOffice.UserService.RestRequests;
+
+namespace LT.DigitalOffice.UserService.Mappers
+{
+    public class EditUserRequestToDbUserMapper : IMapper<EditUserRequest, DbUser>
+    {
+        public DbUser Map(EditUserRequest request)
+        {
+            if (request == null)
+            {
+                throw new ArgumentNullException(nameof(request));
+            }
+
+            return new DbUser
+            {
+                Id = request.Id,
+                Email = request.Email,
+                FirstName = request.FirstName,
+                LastName = request.LastName,
+                MiddleName = request.MiddleName,
+                Status = request.Status,
+                PasswordHash = (new SHA512Managed().ComputeHash(Encoding.Default.GetBytes(request.Password))).ToString(),
+                AvatarFileId = null,
```

Comment:
Whenever a user is updated, their avatar disappears?

Reply:
Fixed